src/pages/Spawn.tsx

```diff
@@ -47,10 +48,12 @@ const SpawnTabs = () => {
         </SideNavGroup>
       </SideNav>
       <div>
-        <Banners
-          banners={bannersState}
-          removeBanner={dispatchBanner.removeBanner}
-        />
+        <Container>
```

Comment:
adds margin left to banner

Reply:
There is now a horizontal scroll on the page which shouldnt exist. 
![scroll](https://user-images.githubusercontent.com/4605522/94854481-f4480500-03fa-11eb-9b7b-211dff57afb0.gif)